### ISSUE TITLE
Fetch metadata: add resource key option

### ISSUE THREAD

**Issue by TA40** [iterative/PyDrive2#127]
I couldn't find anything in the issues/in the code, but is this implemented already?
Would be nice to provide the resource key after fetching Metadata.
https://developers.google.com/drive/api/v3/resource-keys

**Comment by shcheklein** [comment]
I think fetch metadata by default gets all the available fields (we pass `*` here https://github.com/iterative/PyDrive2/commit/e08de92e3299f8d117351ef0817a226fb284ac63#diff-6aa1711e5d0f959a45172852754b9e4569aedc7563d239b63228f7eea85f1cfeR433 )

Could you try to run it? Do you see that we need to specify some fields explicitly?

I would first test this using the online API sandbox: https://developers.google.com/drive/api/v3/reference/files/get (the `Try it` button to the right) and would also compare with the https://developers.google.com/drive/api/v2/reference/files/get (v2 vs v3) . This way we can understand better what do we need here.

**Comment by TA40** [comment]
Well, I tried fetching Metadata (fields=*) for v2 and v3 in the links you provided and Google tells me the following at the end of the response:
```json
"linkShareMetadata":{

    "securityUpdateEligible":false,
    "securityUpdateEnabled":true

}
```
but no resourceKey in the whole response.

I tried it with 3 accounts, each with a shared file and a owned file, 2 of them being Google Workspace Accounts, with old and new files. But no resource key in the response yet, I think. 

Maybe I asked a bit too early for this, let's wait and see what the update in september does.
